### ISSUE TITLE
Cache invalidation is missing

### ISSUE THREAD

**Issue by kremalicious** [oceanprotocol/df-web#263]
Whenever new version is deployed it seems there is no cache busting or anything going on. Not even manually reloading gives me new versions. In one browser tab I still have the version from 2 days ago and no easy way of getting latest version except for nuking all my browser's cache. This is not feasible and will lead to crazy amount of support requests

**Comment by KatunaNorbert** [comment]
Good catch. I was keeping this in a ticket for some time inside #135, I also looked into it once but wasn't finding something relevant on Svelte side, then just got out off radar since df was shot down and we were building mainly locally.

I'll appreciate If you could leave me some leads on this, thanks!

**Comment by kremalicious** [comment]
Maybe something from here?

- https://stackoverflow.com/questions/64552412/how-to-do-cache-busting-in-rollup-js
- https://stackoverflow.com/questions/60115750/hashed-file-names-using-rollup

**Comment by KatunaNorbert** [comment]
Thank you @kremalicious! 
Also great from you to list the goals to aim for during the build.

**Comment by idiom-bytes** [comment]
I have tested this branch/PR both locally, via docker, and it works. Also, i have reviewed the basic principles/implementation of the caching mechanisms by ranodmly generating a hash for each build, and I can see it in the output when i build docker. 

I'm going to deploy this based on worries WRT stabilization, and we can improve more of it later if needed.
Hopefully this helps stabilize some of these updates we're deploying pre-weekend.

Thanks a lot for the ticket, and guidance on this @kremalicious.
I asked you to review the PR and to please create more tickets if needed to refine/improve.

Cheers!
